### ISSUE TITLE
[BUG] Not all Kvantum themes are showing

### ISSUE THREAD

**Issue by koruninn** [baduhai/Koi#98]
**Describe the bug**
i have the orchis theme installed, kvantum recognises both light and dark variants but koi only recognises the light kvantum theme. i even reinstalled the theme and rebooted the system and the issue persists. i'm attaching screenshots to compare how the list looks in both kvantum and koi

**To Reproduce**
1. open koi
2. click on preferences
3. enable the kvantum style toggle
4. expand the dark theme list

**Expected behavior**
show the full list of orchis themes

**Screenshots**
![Captura de pantalla_20241027_204214](https://github.com/user-attachments/assets/109f8672-2479-4a42-8066-724d2af052d4)
![Captura de pantalla_20241027_204121](https://github.com/user-attachments/assets/a54ce89c-f84c-4db0-a9c4-9ff290a6fa56)

**Desktop (please complete the following information):**
 - Distro: nobara 40
 - Koi Version: 0.3.1
 - Plasma Version: 6.2
 - Intallation method: rpm package

**Comment by MartinVonReichenberg** [comment]
Hello, 

As of now, we cannot control every single Kvantum theme which would automatically work within Koi because they are way too specific and robust to debug each therefore some may work, others might not.
All in all I, myself, am unable to find a solution as it is beyond my scope of what I am able to do when it comes to coding as such, thus writing - "you are on your own".

**Comment by pc-v2** [comment]
probably it's on the same folder, you have to make it separate

**Comment by koruninn** [comment]
i'm well aware themes have to be in separate folders, i'm afraid is an issue of the theme itself

**Comment by pc-v2** [comment]
dark and light theme must be in separate folder
here's the default folder tree for orchis
![image](https://github.com/user-attachments/assets/dff7fe25-49b9-4f77-b436-6c6b82645af4)

it should be 
- Kvantum/Orchis
- Kvantum/OrchisDark
- Kvantum/Orchis-solid
- Kvantum/Orchis-solidDark

separate the light and dark to two folder, then move the .svg and .kvconfig according to their name.

Here's what I do to WhiteSur
![image](https://github.com/user-attachments/assets/c160a3a1-9783-4214-8295-5eee32af1965)
![image](https://github.com/user-attachments/assets/6917524e-ce0b-4290-8b05-c0d79b22ae8d)

**Comment by koruninn** [comment]
yeah that's what i did but that doesn't seem to work at all, i don't really have an issue with doing it manually but i thought it was worth pointing it out. thanks for your time anyway!

**Comment by pc-v2** [comment]
> yeah that's what i did but that doesn't seem to work at all, i don't really have an issue with doing it manually but i thought it was worth pointing it out. thanks for your time anyway!

try restart koi, it should redetect

**Comment by MartinVonReichenberg** [comment]
Still no improvements over this matter in Koi [0.4](https://github.com/baduhai/Koi/releases/tag/0.4) ?_?

**Comment by koruninn** [comment]
it's been so long i totally forgot about it, got used to change the kvantum themes manually

**Comment by MartinVonReichenberg** [comment]
@koruninn 
Still nothing?

**Comment by koruninn** [comment]
still nothing, it only shows the light theme but still has problems showing the dark version

**Comment by XenHat** [comment]
I have the same issue with "WhiteSur":
Kvantum theme list:

![Image](https://github.com/user-attachments/assets/78fe696e-a27f-476c-b12b-3a72e5315817)

Koi list:

![Image](https://github.com/user-attachments/assets/62dfd2bc-7d02-42fa-a0ef-20769b742ea6)

**Comment by MartinVonReichenberg** [comment]
Hello @XenHat,

There is possible fix in #105 , I shall check it out and tag a new release accordingly soon . . .

**Comment by MartinVonReichenberg** [comment]
This issue #98 solved via **Pull Request** #105.

_Closing as complete ..._
